<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTimerRem</class>
 <widget class="QDialog" name="SetTimerRem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>470</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>206</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;取消&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="_src.qrc">
     <normaloff>:/new/prefix1/gp_source/close_2.png</normaloff>:/new/prefix1/gp_source/close_2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>47</width>
     <height>47</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>206</y>
     <width>1</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="yes">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>206</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>確認添加
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="_src.qrc">
     <normaloff>:/new/prefix1/gp_source/addCalendar.png</normaloff>:/new/prefix1/gp_source/addCalendar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>49</width>
     <height>49</height>
    </size>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>102</y>
     <width>101</width>
     <height>38</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;清松手寫體2&quot;;
background-color: rgb(255, 255, 255, 0%);
border: none ;


</string>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>hh:mm</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>41</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>211</width>
     <height>41</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;清松手寫體2&quot;;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="sup_View">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>101</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;清松手寫體2&quot;;


</string>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::AmPmSection</enum>
   </property>
   <property name="displayFormat">
    <string>a</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-12</x>
     <y>-10</y>
     <width>480</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>311</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>311</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>470</width>
     <height>311</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255, 40%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CustomLabel" name="switch_sup">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>100</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>151</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>8</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>241</width>
     <height>91</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/gp_source/timeRemainder.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>0</y>
     <width>20</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>0</y>
     <width>20</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>268</y>
     <width>470</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-7</y>
     <width>470</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="remainder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>80</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>251</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;清松手寫體2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>155</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>21</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>161</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;清松手寫體2&quot;;</string>
   </property>
   <property name="text">
    <string>提醒:</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>line_2</zorder>
  <zorder>sup_View</zorder>
  <zorder>cancel</zorder>
  <zorder>frame</zorder>
  <zorder>yes</zorder>
  <zorder>dateEdit</zorder>
  <zorder>timeEdit</zorder>
  <zorder>switch_sup</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>line_4</zorder>
  <zorder>remainder</zorder>
  <zorder>label_2</zorder>
  <zorder>line_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="_src.qrc"/>
 </resources>
 <connections/>
</ui>
